### ISSUE TITLE
Гугл-документ

### ISSUE THREAD

**Issue by Kirill1371** [Kirill1371/Kup_3_sem#12]

**Comment by Kirill1371** [comment]
Открыл Вам доступ (sapr91@mail.ru)

https://docs.google.com/document/d/1RLnnUTFvlGLpORQLSjQSk1gcvl8lTlhWStx4KiWaPXo/edit?usp=drivesdk

**Comment by wldmr-key** [comment]
нет, у меня нет эккаунта такого на гугле. Или сделай общий доступ или кидай, уж, сюда

**Comment by Kirill1371** [comment]
Открыл доступ 

https://docs.google.com/document/d/1RLnnUTFvlGLpORQLSjQSk1gcvl8lTlhWStx4KiWaPXo/edit?usp=drivesdk

**Comment by wldmr-key** [comment]
Введение - убрать первое и второе лицо
Остальной текст (разберем грндфитнес) - тоже
Концептуальная схема - неправильная
Логическая схема - неправильная

В общем - пока ужас. Исправь в начале все замечания по текущим задачам

**Comment by Kirill1371** [comment]
Обновил документ с РПЗ в соответствии с согласованными исправлениями

https://docs.google.com/document/d/1Q2lAGHNPW7xRn6fvx_kZPrUW_KHPjCc7j5fXSKWIIXE/edit?usp=sharing

**Comment by wldmr-key** [comment]
разберись с моделями

**Comment by Kirill1371** [comment]
Финальный вариант (список литературы добавлю завтра). Подскажите, нужно ли комментировать скриншоты системы и добавить ли код моделей БД в конце РПЗ в приложении?
https://docs.google.com/document/d/12i_49z1vYpJvvkXGe8Le_MKxLaOSfmUr3egStBLfwmg/edit?usp=sharing

**Comment by Kirill1371** [comment]
А также изменённый порядок развёртывания системы:

- Арендовать сервер (минималку)
- Скопировать на сервер части сайта (бек, фронт)
- Установить на сервер NodeJS и необходимые пакеты
- Добавить IP сервера в белый список  MongoDB
- Купить доменное имя вида mysite.ru и указать в качетве A записи IP сервера

- Так как у нас 2 инстанса nodejs на разных портах 3000, 5000, а нам нужно на выходе получить 80 и 443 (http, https) то будем использовать веб-сервер nginx в качестве reverse-proxy
- Установить на сервер nginx
- Указать nginx перенаправлять запросы для mysite.ru на http://localhost:3000, а для api.mysite.ru на http://localhost:5000
- Подключить автоматическую генерацию сертификата от LetsEncrypt для доменов, например через CertBot (для работы HTTPS)

- Изменить во фронте адрес к беку на адрес в интернете api.mysite.ru
- Запустить проект выполнив npm run prod (бек, фронт)
